Removed modifier support for `no-route-action` rule

thiagofesta opened the pull request:
[ember-route-action-helper](https://github.com/DockYard/ember-route-action-helper) doesn't have `route-action` modifier. In this case, removed modifier support for `no-route-action` rule.
This fixes issue raised on: https://github.com/ember-template-lint/ember-template-lint/pull/2030#issuecomment-896931655

Also removed binds for class methods and instead of just call them normally.

elwayman02 reviewed (approved):
Thanks for the quick follow up on this!

thiagofesta commented:
@bertdeblock and @scalvert I am addressing your comments from https://github.com/ember-template-lint/ember-template-lint/pull/2030

Let me know what you think. Thanks

scalvert commented:
Wow, that was fast. Thanks! And thanks, @bertdeblock, for catching that modifier sneaking in.
